mdoc-js/src/main/scala/mdoc/modifiers/JsMods.scala

```diff
@@ -49,4 +53,19 @@ object JsMods {
       }
     }
   }
+
+  def parse(fences: Text, info: Input, reporter: Reporter): AllMods = {
+    val jsMods = parse(info, reporter)
+
+    if( reporter.hasErrors ) {
+      AllMods(jsMods, Array[String]())
+    } else {
+      // We already know that the scala fences are valid.
+      val fenceBlocks = fences.value.split(" ")
+      val dropScala = fenceBlocks.tail // first entry is scala
+      val remainingFences = dropScala.filterNot(_.contains("mdoc:js"))
+      // println(remainingFences.mkString(" ", " ", ""))
```

Comment:
```suggestion
      val remainingFences = dropScala.filterNot(_.contains("mdoc:"))
```
would be a bit more general?

Reply:
Although I think it _could_ be, in this case, I _think_ it will only get called when processing Js markdown, as it's in `JsMods`. Further the intent of the PR, extra tests and entirety of thought invested was scala JS only. In a way, this  defends that assumption. 

I'm  unclear if this would be a dangerous change or not.  From what I can tell, unit tests pass if I do make this change. Instinctively, I'm inclined to try to avoid unanticipated effects and leave as is - but I'll follow your lead on it.

Reply:
I think this should be fine.